### PR TITLE
Fix go.mod import names and che-machine-exec Readme

### PR DESCRIPTION
**Pull request by flacatus**
This PR sync go modules with new eclipse-che org
Signed-off-by: Flavius Lacatusu <flacatus@redhat.com>

### CODE REVIEW

File: api/websocket/connect.go [hunk 1]
```diff
@@ -15,11 +15,11 @@ package websocket
 import (
 	jsonrpc "github.com/eclipse/che-go-jsonrpc"
 	"github.com/eclipse/che-go-jsonrpc/jsonrpcws"
-	"github.com/eclipse/che-machine-exec/api/events"
-	execRpc "github.com/eclipse/che-machine-exec/api/jsonrpc"
-	"github.com/eclipse/che-machine-exec/api/model"
-	"github.com/eclipse/che-machine-exec/auth"
-	"github.com/eclipse/che-machine-exec/common/rest"
+	"github.com/eclipse-che/che-machine-exec/api/events"
+	execRpc "github.com/eclipse-che/che-machine-exec/api/jsonrpc"
```
Comment (AndrienkoAleksandr) [inline api/websocket/connect.go:19]: Don't edit jsonrpc lib import. It was not migrated to the new repo.

**Review by sleshchenko — APPROVED** [review]
I haven't tested but did not catch any issue with a quick look

**Comment by flacatus** [comment]
cc @tolusha @AndrienkoAleksandr

**Comment by flacatus** [comment]
@sleshchenko unit tests build the che-machine-exec binary. I guess with that will be enough test

**Review by tolusha — APPROVED** [review]

**Review by AndrienkoAleksandr — APPROVED** [review]

**Pull request merged**